content/en/blog/_posts/2017-04-00-Multi-Stage-Canary-Deployments-With-Kubernetes-In-The-Cloud-Onprem.md

```diff
@@ -3,6 +3,8 @@ title: " How Bitmovin is Doing Multi-Stage Canary Deployments with Kubernetes in
 date: 2017-04-21
 slug: multi-stage-canary-deployments-with-kubernetes-in-the-cloud-onprem
 url: /blog/2017/04/Multi-Stage-Canary-Deployments-With-Kubernetes-In-The-Cloud-Onprem
+author: >
+  Daniel Hoelbling-Inzko (Bitmovin)
 ---
 _Editor's Note: Today’s post is by Daniel Hoelbling-Inzko, Infrastructure Architect at Bitmovin, a company that provides services that transcode digital video and audio to streaming formats, sharing insights about their use of Kubernetes._  
```

Comment:
I'd drop this note. We now just credit the person and optionally their organization.